requirements.txt

```diff
@@ -87,7 +87,8 @@ geonode-pinax-notifications==6.0.0.2
 # GeoNode org maintained apps.
 # django-geonode-mapstore-client==4.0.5
 git+https://github.com/GeoNode/geonode-mapstore-client.git@master#egg=django_geonode_mapstore_client
-git+https://github.com/GeoNode/geonode-importer.git@master#egg=geonode-importer
+#git+https://github.com/GeoNode/geonode-importer.git@geonode_12124_assets#egg=geonode-importer
+git+https://github.com/GeoNode/geonode-importer.git@ISSUE_247#egg=geonode-importer
```

Comment:
@mattiagiupponi this must be changed to master before merging.

Reply:
i'll switch them to master since the assets are merged